modules/core/src/main/java/io/fluo/core/oracle/Stamp.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2014 Fluo authors (see AUTHORS)
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.fluo.core.oracle;
+
+public class Stamp {
+  private final long stamp;
+  private final long gcStamp;
+
+  Stamp(long stamp, long gcStamp) {
+    this.stamp = stamp;
+    this.gcStamp = gcStamp;
+  }
+
+
+  public long getTimestamp() {
```

Comment:
you could rename this to `getTxTimestamp()` to differentiate between `getGcTimestamp()`